compiler/rustc_privacy/src/errors.rs

```diff
@@ -73,6 +73,8 @@ pub struct InPublicInterface<'a> {
     pub descr: DiagnosticArgFromDisplay<'a>,
     #[label(privacy_visibility_label)]
     pub vis_span: Span,
+    #[suggestion(code = "", applicability = "maybe-incorrect")]
```

Comment:
This suggestion isn't actually modifying anything

Reply:
I am not sure what it should modify and how to do it